src/test/java/org/shikshalokam/backend/scp/SelfCreationPortalCreateNewPermission.java

```diff
@@ -33,11 +33,10 @@ public void init() {
 
         // Log the extracted token
         logger.info("Internal Access Token: " + internalAccessToken);
-        logger.info(response.prettyPrint());
     }
 
-    @Test(description = "Verifies the functionality of creating new user's permission.")
-    public void testCreateNewPermission() {
+    @Test(description = "Verifies the functionality of creating new user's permission with valid payload.")
+    public void testCreatePermissionWithValidPayload() {
         logger.info("Started calling the CreatePermission API:");
         try {
             RestAssured.baseURI = PropertyLoader.PROP_LIST.get("scp.qa.api.base.url").toString();
```

Comment:
This already we are doing in the base call hence it is not required again here . Remove this